### DISCUSSION TITLE
Unable to use copilot's services

### DISCUSSION THREAD

**Discussion by AlgoOy** [community/community#114893] [GitHub Education]
### Hi everyone,

I don't know why, but I haven't been able to access the services offered by Co-Pilot. In fact, I have applied for it for a long time. I also received an email after that that my application was approved, but after all this time I still can't access copilot. In the IDE, whether it is VS code or Jetbrains tools, copilot cannot be used normally.

post my ticket: https://support.github.com/ticket/personal/0/2685998

![image](https://github.com/community/community/assets/44948890/a42ae597-c856-41a7-81af-91b2070e8787)
![image](https://github.com/community/community/assets/44948890/191e4778-6350-4e37-a938-0a521de2d60d)
![image](https://github.com/community/community/assets/44948890/1937a4f1-1141-47f1-8514-e60856ed1fcd)

**Comment by samus-aran** [comment]
Hi @AlgoOy 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
